### ISSUE TITLE
Pass ArrayBuffer to webview c-side function?

### ISSUE THREAD

**Issue by dbadb** [Tracktion/choc#54]
When I pass an ArrayBuffer to a webview-bound function (ie via webview->bind("myfunc", []...), choc declares the value to be an object.  The object's type is reported as "" and the object has no named members.  I see special code in the javascript V8 binding for ArrayBuffers, but this appears to be irrelevant for webview transfers since JSON.stringify is the serialization mechanism in play. 

I think there are two potential workarounds: 1. convert the ArrayBuffer to an array, 2: use http post. #1 is certainly not-so-efficient and I've yet to explore #2 fwiw.

I'm guessing this is a known limitation but thought I'd submit this issue to place it on radar screens.

**Comment by julianstorer** [comment]
Yes, I was leaving that one until later, as I'm also not sure the best plan!

**Comment by dbadb** [comment]
Here's a related thread I stumbled across - hadn't heard that Photoshop was webview-ing...

https://forums.creativeclouddeveloper.com/t/webview-postmessage-with-arraybuffer-terribly-slow/7055/12
